screenpy/actor.py

```diff
@@ -159,13 +202,29 @@ def cleans_up(self) -> None:
         self.cleans_up_ordered_tasks()
 
     def exit(self) -> None:
-        """Direct the Actor to forget all their Abilities."""
+        """Direct the Actor to forget all their Abilities.
+
+        Aliases:
+            * :meth:`~screenpy.actor.Actor.exit_stage_left`
+            * :meth:`~screenpy.actor.Actor.exit_stage_right`
+            * :meth:`~screenpy.actor.Actor.exit_through_vomitorium`
+        """
         self.cleans_up()
         for ability in self.abilities:
             ability.forget()
         self.abilities = []
 
-    exit_stage_left = exit_stage_right = exit_through_vomitorium = exit
+    def exit_stage_left(self) -> None:
+        """Alias for :meth:`~screenpy.actor.Actor.exit`."""
+        return self.exit()
+
+    def exit_stage_right(self) -> None:
+        """Alias for :meth:`~screenpy.actor.Actor.exit`."""
+        return self.exit()
+
+    def exit_through_vomitorium(self) -> None:
+        """Alias for :meth:`~screenpy.actor.Actor.exit`."""
+        return self.exit()
```

Comment:
Of course these aliases are strictly essential and must be preserved at all costs